src/pages/ProfileEdit.vue

```diff
@@ -60,25 +59,32 @@ q-page.fit.row.wrap.justify-center
           maxlength='30'
         )
         q-input.q-mt-sm(
-          outlined,
           label='Bio',
           placeholder='Bio',
           v-model='bio',
           bg-color='white',
           counter,
+          dense,
+          rows='5',
           maxlength='140',
           type='textarea'
         )
     //- Add save button in the middel
     q-card-section
       .row.justify-center.items-center
-        q-btn(@click='saveProfile', color='primary', label='Save')
+        q-btn(@click='saveProfile', flat, label='Save', border='none')
 </template>
 
 <style lang="sass" scoped>
 .edit-card
   width: 100%
   max-width: 30rem
   height: 100%
-  max-height: 40rem
+  max-height: 35rem
+
+.text-subtitle2
+  color: gray
+
+.text-subtitle1
+  font-size: 25px
 </style>
```

Comment:
Please move these changes to a different branch to keep this pull-request specific to the navbar

Reply:
Noted.